python/cudf/cudf/_lib/io/CMakeLists.txt

```diff
@@ -0,0 +1,29 @@
+# =============================================================================
+# Copyright (c) 2022, NVIDIA CORPORATION.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except
+# in compliance with the License. You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software distributed under the License
+# is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+# or implied. See the License for the specific language governing permissions and limitations under
+# the License.
+# =============================================================================
+
+set(cython_sources datasource.pyx utils.pyx)
+set(linked_libraries cudf::cudf)
+rapids_cython_create_modules(
+  SOURCE_FILES "${cython_sources}" LINKED_LIBRARIES "${linked_libraries}" MODULE_PREFIX io_ CXX
```

Comment:
Style wise can we lead with `CXX` instead of having it at the end. That would better match the order of documented options to `rapids_cython_create_modules`